tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -723,6 +723,7 @@ def convert_pytorch_to_onnx(model, input_shape, opset_version, example_inputs, o
     torch.onnx.export(model,
                       inputs,
                       model_onnx,
+                      operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
```

Comment:
@popovaan, @eaidova, is it possible to add one small PyTorch model with unsupported operation please?

Reply:
Added test with small custom op, which checks that conversion to ONNX step passes.